### PR TITLE
adding proportional-shrink scripts to AMI-SDM

### PR DESCRIPTION
**Pull request by GaofengCheng**

### CODE REVIEW

**Comment by GaofengCheng** [comment]
Related to #1627

**Comment by danpovey** [comment]
Thanks!  These are very nice improvements.  If you ever get a chance to do something similar in the Switchboard s5c recipe or the tedlium s5_r2 recipe, that would be great too.  Not urgent.

File: egs/ami/s5b/local/chain/tuning/run_tdnn_1f.sh [hunk 1]
```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+# same as 1e but uses batchnorm components instead of renorm also adding
+# proportional-shrink 10, trained with 4 epochs
+
+# System               tdnn1e_sp_bi_ihmali tdnn1f_sp_bi_ihmali
```
Comment (danpovey) [inline egs/ami/s5b/local/chain/tuning/run_tdnn_1f.sh]: actually I just realized that you don't specify what microphone these results are with.  I assume sdm, but you should probably state that, and say with what command line options this script was run.

Comment (GaofengCheng) [inline egs/ami/s5b/local/chain/tuning/run_tdnn_1f.sh]: @danpovey Oh..... after the SWBD scripts are OK, I'll adding the command lines in these scripts together with the SWBD scripts

**Pull request merged**